lib/model/query/projects.js

```diff
@@ -94,20 +94,26 @@ ${extend|| sql`
     on field_keys."projectId"=projects.id`}
 ${(actorId == null) ? sql`` : sql`
 inner join
-  (select id from projects
+  (select id, min("roleId") as "bestRole" from projects
     inner join
-      (select "acteeId" from assignments
-        inner join (select id from roles where verbs ? 'project.read') as role
+      (select "acteeId", role.id as "roleId", role.verbs from assignments
+        inner join (select id, verbs from roles where verbs ? 'project.read') as role
```

Comment:
Ooh, good point. How about I check for both `project.read` (because this is a project-oriented endpoint) _and_ `form.list` (because it is functionally getting a list of forms).

I can make the change to ` ?& array['project.read', 'form.list']` documented here: https://www.postgresql.org/docs/9.4/functions-json.html#FUNCTIONS-JSONB-OP-TABLE